### PR TITLE
3 persona docker

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Updates the docker-compose file to include the 3 personas used for the demo. I've rmeoved the old single persona one as we don't use it for anything as far as I can tell.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,105 +1,32 @@
-version: '3'
+#docker compose -f docker-compose-3-persona.yml up --build -d
```
Comment (dc-andysign) [inline docker-compose.yml:1]: Sorry, is this perhaps a leftover ?

Would it look nicer for this line to be like the line below ?

```
docker compose up -d # OR docker compose down -v --remove-orphans && docker compose up -d
```

**Review by dc-andysign — CHANGES_REQUESTED** [review]
Awesome .. one small thing ..

**Review by dc-andysign — APPROVED** [review]
👍

**Comment by rmlearney-digicatapult** [comment]
Rename from `docker-compose.yml`?

**Comment by mattdean-digicatapult** [comment]
The reason to use `docker-compose.yml` is you can then just do 

```
docker compose up -d
```

to get a detatched infrastructure. As this is the stand I'd strongly recommend sticking with it

**Pull request merged**